Controller/RedirectRouteImportController.php

```diff
@@ -46,14 +48,30 @@ public function __construct(FileImportInterface $import, $importPath)
         $this->importPath = $importPath;
     }
 
+    /**
+     * {@inheritDoc}
+     */
+    public function getSecurityContext()
+    {
+        return RedirectAdmin::SECURITY_CONTEXT;
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    public function getLocale(Request $request)
+    {
+        return $request->get('locale', null);
+    }
+
     /**
      * Import file which was uploaded.
      *
      * @param Request $request
      *
      * @return Response
      */
-    public function importAction(Request $request)
+    public function postAction(Request $request)
```

Comment:
Find this a little bit strange https://github.com/sulu/sulu/blob/6176cd69bb00c5b821c1bf08c987663fa493191b/src/Sulu/Bundle/SecurityBundle/EventListener/SuluSecurityListener.php#L66-L70

/cc @danrot @chirimoya

Reply:
It was (and probably still is) the most convenient method to handle that for the end user :slightly_smiling_face: Otherwise the configuration gets a lot more complex I guess (how do you tell which permission should be taken?).

Reply:
@danrot I'm confused in which case a POST is an Edit and not an Add in our case?

Reply:
Everytime an action is triggered, e.g. locking a user or moving a collection. Most of the time these functions are called `postTriggerAction` (although I am not really happy with that :see_no_evil:), so you could search our code base for it.